backfill/gormstore.go

```diff
@@ -118,46 +119,108 @@ func (s *Gormstore) EnqueueJob(repo string) error {
 	return nil
 }
 
-func (s *Gormstore) BufferOp(ctx context.Context, repo, kind, path string, rec *typegen.CBORMarshaler, cid *cid.Cid) (bool, error) {
-	s.lk.RLock()
-
-	// If the job doesn't exist, we can't buffer an op for it
-	j, ok := s.jobs[repo]
-	s.lk.RUnlock()
-	if !ok {
-		return false, ErrJobNotFound
-	}
-
+func (j *Gormjob) BufferOps(ctx context.Context, since *string, rev string, ops []*bufferedOp) (bool, error) {
 	j.lk.Lock()
 	defer j.lk.Unlock()
 
 	switch j.state {
 	case StateComplete:
-		return false, ErrJobComplete
-	case StateInProgress:
-	// keep going and buffer the op
-	default:
 		return false, nil
+	case StateInProgress, StateEnqueued:
```

Comment:
Why are we buffering ops on enqueued backfill jobs here? They won't have been checked out yet if they're still enqueued so any events they get will be included in the checkout, right?

Reply:
ah, thats true